src/Sylius/Behat/Context/Setup/CartContext.php

```diff
@@ -141,19 +143,21 @@ private function pickupCart(): string
         $channel = $this->sharedStorage->get('channel');
         $channelCode = $channel->getCode();
 
-        $commandPickupCart = new PickupCart($tokenValue);
-        $commandPickupCart->setChannelCode($channelCode);
-
         if ($this->sharedStorage->has('token') && $this->sharedStorage->has('user')) {
             $user = $this->sharedStorage->get('user');
 
             if ($user instanceof ShopUserInterface) {
                 /** @var CustomerInterface $customer */
-                $customer = $user->getCustomer();
-                $commandPickupCart->setEmail($customer->getEmail());
+                $email = $user->getCustomer()->getEmail();
             }
         }
 
+        $commandPickupCart = new PickupCart(
```

Comment:
```suggestion
        $pickupCart = new PickupCart(
```
just wondering becouse above there is also `$updateCart` rather than `$commandUpdateCart`